korman/properties/modifiers/gui.py

```diff
@@ -633,6 +653,30 @@ def _create_moul_nodes(self, clickable_object, nodes, linkingnode, age_name, clk
         linking_panel_name.value = self.link_destination if self.link_destination else self.age_name
         linking_panel_name.link_output(linkingnode, "pfm", "TargetAge")
 
+        # Share MSB
+        if self.shareable:
+            # Region
+            share_region = nodes.new("PlasmaVolumeSensorNode")
```

Comment:
This will collide with the function argument named `share_region`. Whoops!

Reply:
I renamed it to `share_rgn` in the `_create_moul_nodes` def.